### ISSUE TITLE
SVG accessibility

### ISSUE THREAD

**Issue by vicbergquist** [melsicon/melsicon.de#17]
### **Type of Issue**

- [ ] Bug
- [X] Enhancement
- [ ] Documentation
- [ ] Maintenance
- [ ] Question

#### Priority

- [X] Low
- [ ] Medium
- [ ] High
- [ ] Critical

## **Description**

SVGs are missing important accessibility features like role, aria-labelledby, as well as title and desc tags
